alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunnerIT.kt

```diff
@@ -65,7 +65,7 @@ class MonitorRunnerIT : AlertingRestTestCase() {
     fun `test execute monitor returns search result`() {
         val uniqueName = "unique name"
         val query = QueryBuilders.termQuery("monitor.name.keyword", uniqueName)
-        val input = SearchInput(indices = listOf("*"), query = SearchSourceBuilder().query(query))
+        val input = SearchInput(indices = listOf(".*"), query = SearchSourceBuilder().query(query))
```

Comment:
Is this the way to search for hidden indices? `*` does not match but `.*` does match? Do you have a link to any docs on searching hidden indices?

Reply:
Here is a reference, in the PR of introducing the hidden index into Elasticsearch: https://github.com/elastic/elasticsearch/pull/51164,

> Wildcard expansion will not return hidden indices by default unless
the wildcard pattern begins with a .. This behavior is similar to
shell expansion of wildcards.